backend/api/v1/endpoints/chatbot/huggingface.py

```
@@ -20,64 +23,41 @@
-model = "mistralai/Mistral-7B-Instruct-v0.3"
-llm = HuggingFaceEndpoint(
-    model=model,
-    task="text-generation",  # Explicitly specify the task"
-    max_new_tokens=200,
-    temperature=0.7,
-    huggingfacehub_api_token=hf_key
-)
+gemini_model = genai.GenerativeModel("gemini-2.0-flash")
 
 
 import fitz  # PyMuPDF
```

Ruff (0.8.2)

Module level import not at top of file

(E402)